test/puppetlabs/ring_middleware/core_test.clj

```diff
@@ -118,6 +146,48 @@
            (add-ring-handler proxy-webserver# ~proxy-handler ~endpoint))
          ~@body)))
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;; Core Helpers
+
+(deftest json-response-test
+  (testing "json response"
+    (let [source {"key1" "val1", "key2" "val2"}
```

Comment:
json-response looks like it will turn keywords into string keys; agreed that this should be using keyword keys / newline / no comma